### ISSUE TITLE
Commonly Used Fonts

### ISSUE THREAD

**Issue by ghost** [GitSquared/mrworldwide#2]
Could there be a "search bar" with results of tried and tested fonts that are commonly used, so that the user does not have to download those fonts to try them on your website?

**Comment by GitSquared** [comment]
Storing results of commonly tested fonts would require turning this offline tool into a full web app, which might be a bit overkill. However, building on the idea of a "search bar", perhaps there could be one to search Google Fonts (or other services) and the website could download the font transparently for you.

**Comment by ghost** [comment]
Well I was thinking maybe there is a `.JSON` or `.JS` file (or few) that contain the results so all it has to do is load it up and display the data on the map without having to do any of that.

_Edit:_
And by results I mean we can "pre-define" them by manually finding out ourselves and logging the outcome.

**Comment by GitSquared** [comment]
Unfortunately there is no such dataset afaik, so i'd have to build it myself.
Storing test results so they can be retrieved by future users would require a web app - mostly a backend storage like a database or something.

**Comment by ghost** [comment]
Actually, if you have an object or array in a `.js` file, and you put the `<script>` tag in the index, it can work like a client-side database! [I'm doing that for one of my projects I'm working on currently](https://github.com/ForgivenNin/javacrypt/blob/master/cipher/corpora.js).

**Comment by GitSquared** [comment]
Indeed, but that doesn't help much unless we have a dataset of popular fonts and their compatibility first. Anyway, I'll look into this, thanks for the suggestion.

**Comment by ghost** [comment]
That's definitely where most of the work would come in.

**Comment by ghost** [comment]
Just to follow up, @GitSquared, do you think this is something that could be done?

**Comment by GitSquared** [comment]
@ForgivenNin Yes, I've kept the issue open to work on it soon™. Unfortunately I haven't had the time to dig into this yet.
